frontend/components/base/base-tabs/baseTabs.spec.js

```diff
@@ -0,0 +1,48 @@
+import { shallowMount } from "@vue/test-utils";
+import BaseTabs from "./BaseTabs";
+
+let wrapper = null;
+const options = {
+  components: { BaseTabs },
+  propsData: {
+    tabs: [
+      { id: "tab1", name: "Tab 1" },
+      { id: "tab2", name: "Tab 2" },
+    ],
+    activeTab: { id: "tab1", name: "Tab 1" },
+  },
+};
+
+beforeEach(() => {
+  wrapper = shallowMount(BaseTabs, options);
+});
+
+afterEach(() => {
+  wrapper.destroy();
+});
+
+describe("BaseTabs", () => {
+  it("on click first tab emit change-tab event with first tab id", async () => {
+    const button = wrapper.findAll(".tab__button").at(0);
```

Comment:
tips for the sentence in the test. Since it's start with 'it' you can replace the description by it('emit change-tab event with first tab id on click in the first tab', ...)